src/components/Teaser/schema.js

```diff
@@ -45,12 +49,20 @@ export const TeaserSchema = (props) => {
         title: intl.formatMessage(messages.Source),
         widget: 'object_browser',
         mode: 'link',
-        selectedItemAttrs: ['Title', 'Description', 'hasPreviewImage'],
+        selectedItemAttrs: [
+          'Title',
+          'headtitle',
```

Comment:
In order for this to work, we need in our projects to add this as metadata in the catalog.

/cc @tisto